docx/oxml/text/symbol.py

```diff
@@ -0,0 +1,51 @@
+# encoding: utf-8
+
+"""
+Custom element classes related to text symbols (CT_Sym).
+"""
+
+from ..xmlchemy import (
+    BaseOxmlElement, OptionalAttribute
+)
+from ..simpletypes import (
+    ST_String
+)
+from warnings import warn
+
+# this font face uses WordPerfect, it maps usual chars to special ones
+map_WP_TypographicSymbols = {
+    '0027': "§",
+    '0040': "”",
+    '0038': "©",
+    '003D': "’",
+    '0041': "“",
+    '0042': "–",
+    '0043': "—"
+}

```

Comment:
What do you think of making this dictionary map fonts to character mapping?
Something like this?
```
font_to_char_mapping_dict: {
    'WP_TypographicSymbols': {
        '0027': "§",
        '0040': "”",
        '0038': "©",
        '003D': "’",
        '0041': "“",
        '0042': "–",
        '0043': "—"
     }
}
```
With this approach, we could remove the hard-coded font name from the method, and that way, when we add a new font in the future, we don't need to change the function but only this dictionary.
What do you think?